### PR TITLE
fix(#1024) - Deploy to One Platform Button enhancements

### LINKED ISSUE #1024: Deploy to One Platform Button enhancements
## Describe the solution you'd like

- Add uploading login in front end

## Describe alternatives you've considered

- None
- 
## Additional context

N/A

### PR DESCRIPTION
**Pull request by hybridx**
### Resolves #1024

# Explain the feature/fix

- Added logic for SPADeployment in home-spa

## Does this PR introduce a breaking change

- No

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Did tests pass?
- [x] Was this feature demo'd and the design review approved?

### CODE REVIEW

File: packages/home-spa/src/res/static/microservices.json [hunk 1]
```diff
@@ -2,35 +2,35 @@
...
-      "link": "/get-started/docs/microservices/authorization-service/",
+      "link": "/get-started/docs/microservices/authorization/",
       "icon": "res/img/auth.svg",
       "imgBgColor": "#F3FAF2"
     },
     {
       "name": "Feedback",
       "description": "Allows users to review and rate services and products provided by teams. Built into a power packed microservice so serve all.",
-      "link": "/get-started/docs/microservices/feedback-service/",
+      "link": "/get-started/docs/microservices/feedback/",
```
Comment (deshmukhmayur) [inline packages/home-spa/src/res/static/microservices.json:12]: The previous one was correct. Just the authorization link was incorrect.
```suggestion
      "link": "/get-started/docs/microservices/feedback-service",
```

File: packages/home-spa/src/res/static/microservices.json [hunk 2]
```diff
@@ -2,35 +2,35 @@
...
       "icon": "res/img/feedback.svg",
       "imgBgColor": "#FAEAE8"
     },
     {
       "name": "Notification",
       "description": "Notification framework for One Platform and also all other SPAs",
-      "link": "/get-started/docs/microservices/notifications-service/",
+      "link": "/get-started/docs/microservices/notifications/",
```
Comment (deshmukhmayur) [inline packages/home-spa/src/res/static/microservices.json:19]:
```suggestion
      "link": "/get-started/docs/microservices/notifications-service",
```

File: packages/home-spa/src/res/static/microservices.json [hunk 3]
```diff
@@ -2,35 +2,35 @@
...
       "icon": "res/img/notification.svg",
       "imgBgColor": "#E7F1FA"
     },
     {
       "name": "User Group",
       "description": "A prebuilt integration with LDAP to authenticate, authorize and get user information.",
-      "link": "/get-started/docs/microservices/user-groups-service/",
+      "link": "/get-started/docs/microservices/user-groups/",
```
Comment (deshmukhmayur) [inline packages/home-spa/src/res/static/microservices.json:26]:
```suggestion
      "link": "/get-started/docs/microservices/user-groups-service",
```

File: packages/home-spa/src/res/static/microservices.json [hunk 4]
```diff
@@ -2,35 +2,35 @@
...
       "icon": "res/img/user.svg",
       "imgBgColor": "#FDF7E7"
     },
     {
       "name": "Search",
       "description": "A unified search API gives you ability to search across all hosted applications on the platform.",
-      "link": "/get-started/docs/microservices/search-service/",
+      "link": "/get-started/docs/microservices/search/",
```
Comment (deshmukhmayur) [inline packages/home-spa/src/res/static/microservices.json:33]:
```suggestion
      "link": "/get-started/docs/microservices/search-service",
```

**Review by deshmukhmayur — CHANGES_REQUESTED** [review]

File: packages/home-spa/.env.example [hunk 5]
```diff
@@ -1,2 +1,3 @@
 API_URL=
 DEPLOY_URL=
```
[the remark this answers is not in the document]
Comment (hybridx) [reply packages/home-spa/.env.example]: Change added

**Review by deshmukhmayur — APPROVED** [review]

**Review by gisumit — APPROVED** [review]

**Pull request merged**